_glaredb/sql-functions/read_mongodb.md

```diff
@@ -26,13 +30,21 @@ read_mongodb(<connection_str>, <database>, <collection>)
 In the following example, a 'users' collection is queryed.
 
 ```sql
-select * from read_mongodb(
-  'protocol=mongodb+srv, user=mongo_user, password=mong_user_password',
-  'my_mongo_database',
-  'users'
-);
+SELECT mdb.* FROM read_mongodb(
+  'protocol=mongodb+srv, user=mdb_user, password=mdb_user_password',
+  'mdbDatabaseName',
+  'collectionName'
+) AS mdb;
 ```
 
-Refer to the [documentation on MongoDB data sources] for more information.
+This exposes the fields from the MongoDB collection as a collection of
+fields with the `mdb.` prefix in a SQL query. Select an alternate
+prefix, or no prefix as you see fit.
```

Comment:
i think the `mdb.*` stuff isn't needed. IMO, it adds a bit more confusion than the original example.

Reply:
I see what you mean, but I think almost all of the utility here (with mongodb's proxy/data sources,) has to do with being able to do joins and interactions of data from more than data source: and to do that, effectively, you need/want to have your fields/projections namespaced, otherwise it's _way more confusing_, and I think this extra 'include a namespace" is something that begins to connect for users the _why you might use this function_ and not just _what it does_. 

This isn't an example that people would copy as such, it's more just a "see this operation in action". I'd be inclined to actually add a more complex projection which would give us an opportunity to explain/declare that there are push-down predicates and projections.